tests/test_vav_turndown_during_reheat.py

```diff
@@ -127,3 +127,77 @@ def test_vav_turndown_during_reheat_untested(self):
 
         self.assertEqual(results, expected_results)
         self.assertEqual(verification_obj.check_bool(), "Untested")
+
+    def test_vav_turndown_during_reheat_all_reheat_untested(self):
+        points = [
+            "reheat_coil_flag",
+            "V_dot_VAV",
+            "V_dot_VAV_max",
+        ]
+
+        timestamp = [
+            datetime(2024, 8, 1, 12, 0, 0),
+            datetime(2024, 8, 1, 13, 0, 0),
+            datetime(2024, 8, 1, 14, 0, 0),
+            datetime(2024, 8, 1, 15, 0, 0),
+        ]
+
+        data = [
+            [True, 350, 620],
+            [True, 370, 620],
+            [True, 360, 620],
+            [True, 380, 620],
+        ]
+
+        df = pd.DataFrame(data, columns=points, index=timestamp)
+
+        verification_obj = run_test_verification_with_data(
+            "VAVTurndownDuringReheat", df
+        )
+        results = list(verification_obj.result)
+        expected_results = [
+            "Untested",
+            "Untested",
+            "Untested",
+            "Untested",
+        ]
+
+        self.assertEqual(results, expected_results)
+        self.assertEqual(verification_obj.check_bool(), "Untested")
+
+    def test_vav_turndown_during_reheat_zero_V_dot_VAV_max_untested(self):
+        points = [
+            "reheat_coil_flag",
+            "V_dot_VAV",
+            "V_dot_VAV_max",
+        ]
+
+        timestamp = [
+            datetime(2024, 8, 1, 12, 0, 0),
+            datetime(2024, 8, 1, 13, 0, 0),
+            datetime(2024, 8, 1, 14, 0, 0),
+            datetime(2024, 8, 1, 15, 0, 0),
+        ]
+
+        data = [
+            [True, 350, 0],
+            [True, 370, 0],
```

Comment:
seems like this is the only difference from the previous test. Should the error log being asserted in the test?

Reply:
Good point.

One thing I don't understand is that when I added the following code at the end of the unit test (line 284), the ```logobs``` is empty, which I don't understand. I'll keep this PR for now to take more time to find the reason. This shouldn't impact the testing.

```Python
with self.assertLogs() as logobs:
    self.assertEqual(
        "ERROR:root:Any `V_dot_VAV_max` value shouldn't be zero.",
        logobs.output[0],
    )
```